### PR TITLE
Add dropout

### PR DESCRIPTION
**Pull request by quentinverlhac**
Since there is overfitting

### CODE REVIEW

File: models/pianocktail_cnn.py [hunk 1]
```diff
@@ -18,24 +18,33 @@ def __init__(self, name=config.ModelEnum.PIANOCKTAIL_CNN.value, subspectrogram_p
...
+        self.dropout3 = Dropout(0.1,name=f"{name}_dropout3")
+
         self.dense2 = Dense(config.NUMBER_OF_EMOTIONS, activation=tf.nn.sigmoid, name=f"{name}_output")
 
     def call(self, inputs, training=False):
         net = self.input_layer(inputs)
         net = self.reshape(net)
         net = self.conv1(net)
         net = self.pooling1(net)
+        net = self.dropout1(net)
```
Comment (Antoine-marchais) [inline models/pianocktail_cnn.py]: droupout must only be called on training, if you don't set the training parameter, you will also get dropout during testing

File: models/pianocktail_cnn.py [hunk 2]
```diff
@@ -18,24 +18,33 @@ def __init__(self, name=config.ModelEnum.PIANOCKTAIL_CNN.value, subspectrogram_p
 
         self.pooling1 = AveragePooling2D(pool_size=2,name=f"{name}_pool1")
 
+        self.dropout1 = Dropout(0.1,name=f"{name}_dropout1")
```
Comment (Antoine-marchais) [inline models/pianocktail_cnn.py]: Not sure about the hard coded 0.1, this usually needs to be cross validated

Comment (quentinverlhac) [reply models/pianocktail_cnn.py]: Sure, it was just a try since we don't have cross validation

File: models/pianocktail_cnn.py (hunk 1 above)
Comment (quentinverlhac) [reply models/pianocktail_cnn.py]: Good catch

File: models/pianocktail_cnn.py (hunk 2 above)
Comment (Antoine-marchais) [reply models/pianocktail_cnn.py]: Or at least validated, you get the point :smile:
